examples/chip-tool/commands/clusters/ModelCommand.cpp

```diff
@@ -36,10 +36,19 @@ CHIP_ERROR ModelCommand::RunCommand()
     }
 
     ChipLogProgress(chipTool, "Sending command to node 0x%" PRIx64, mNodeId);
+
+    CommissioneeDeviceProxy * commissioneeDeviceProxy = nullptr;
+    // The return value is ignored because GetDeviceBeingCommissioned throw no device are found.
```

Comment:
```suggestion
    // The return value is ignored because GetDeviceBeingCommissioned returns error if no device is found.
```
That said, shouldn't we be using that error return, instead of whether `commissioneeDeviceProxy` is null?

Reply:
I think we need to provide a better API from `CHIPDeviceController`.But you are right, I have updated the code to:
`if (CHIP_NO_ERROR == CurrentCommissioner().GetDeviceBeingCommissioned(mNodeId, &commissioneeDeviceProxy))`